x-pack/plugin/src/test/resources/rest-api-spec/test/ilm/30_retry.yml

```diff
@@ -7,7 +7,7 @@ setup:
   - do:
       acknowlege: true
       ilm.put_lifecycle:
-        lifecycle: "my_lifecycle"
+        policy: "my_lifecycle"
```

Comment:
It's a little odd that the value on these is `my_lifecycle` when the noun has been changed to `policy` everywhere else, but that's a nitpick.